.vscode/launch.json

```diff
@@ -5,10 +5,10 @@
       "type": "node",
       "request": "launch",
       "name": "Launch With OSS Index",
-      "args": ["ossi"],
+      "args": ["ossi", "--dev"],
```

Comment:
Figured debug should include all deps by default, since the `read-installed` issues we saw came from a project with a circular dev dep.